### ISSUE TITLE
TYPO3 13.4 - Sitepackage not available to add in TypoScript record (include_Static_file)

### ISSUE THREAD

**Issue by tzeier** [benjaminkott/packagebuilder#83]
In ddev I did a fresh install with TYPO3 13.4 LTS, Bootstrap Package 15 and also installed a sitepacke, generated on sitepackagebuilder.com for v13.4 (Beta). The installation was successful, the sitepackage is listed in the Extensions Module, but when I want to add it to the TypoScript record, it's not found there... 
![grafik](https://github.com/user-attachments/assets/390f8c1d-6dc7-4d49-b03c-fb37e21da7af)

Is this a known bug, You're working on? Or did I miss something?

**Comment by mminyades** [comment]
I can confirm this error or behavior as well. Adjusting the settings in "Site Configuration" (Sites/Site Configuration/) under the "General" tab worked for me. In the "Sets for this Site" option field, you should be able to select the Sitepackage. Once selected, the configuration of the Sitepackage is applied.

It appears that other extensions, such as felogin, are also activated through this setting.

I wish significant changes like this were communicated more clearly.

**Comment by tzeier** [comment]
Yeah, I also thought of that. But if I remove bootstrap package in TypoScript and add my site package and BP in the site settings instead, I get a "No page confugured for type=0" error in the FE. 
When I leave BP in thr TypoScript settings an add my own package in Site settings, the configuration of my own package don't apply...
Did I forget something?

**Comment by mminyades** [comment]
I haven't tried the "Bootstrap Package", I can only report that it worked for me with the "Fluid Styled Content" version.

**Comment by criwe** [comment]
Hi, I´ve got the same issue. Maybe it´s not a bug with the sitepackagebuilder, but with the bootstrap package: https://github.com/benjaminkott/bootstrap_package/issues/1536

**Comment by tzeier** [comment]
I see. Yeah might be...

**Comment by linawolf** [comment]
The site package must be included as site set, you don't need a TypoScript record anymore:

https://docs.typo3.org/permalink/t3sitepackage:minimal-extension-siteset
